simulators/utils/predictive_utils.py

```diff
@@ -94,7 +94,9 @@ def __init__(self,
         # TODO: These distributions will no longer be required once the inferential model comes into play.
 
         # Probability of a legal delivery
-        self.legal_delivery_distribution = sps.bernoulli(p=0.9)
```

Comment:
This section includes tweaks to the predictive simulator which may impact all models potentially